tests/xapi-plugins/plugin_smartctl/test_smartctl.py

```diff
@@ -0,0 +1,19 @@
+import json
+import pytest
+
+# Requirements:
+# From --hosts parameter:
+# - host(A1): first XCP-ng host >= 8.3.
```

Comment:
I should have seen it earlier. @benjamreis should have seen it earlier too: you need to decorate your tests (or a test class containing the tests) with:

`@pytest.mark.usefixtures("host_at_least_8_3")`

This will skip the tests on a host which is < 8.3